src/client/contract-client.js

```diff
@@ -117,7 +117,7 @@ class Client {
         const action = {
           channelID: this.channelID,
           nonce: nonce,
-          blockExpirationNumber: blockExpirationNumber,
+          blockExpirationNumber: blockExpirationNumber.toString(),
```

Comment:
This makes sense, but broke a few of the tests:

/test/client/contract-client.js needs updated at lines 368, 390 and 413 to expect the string value for this field:
```
blockExpirationNumber: '11',
```